hicommonwealth/commonwealth
# Fix thread reaction count update

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/7160

## Description of Changes
Fix thread reaction count update on `/discussion` page

## "How We Fixed It"
The updated frontend cache for reaction was not getting used in `<DiscussionPage/>`, this pr reverts some trpc changes that removed cache usage.

## Test Plan
1. Visit any community
2. On the `/discussions` page, react to any thread, verify that reaction count updates
3. Now go to details page of thread (click on the thread) and verify you see the correct reaction count.
4. Repeat steps 2 and 3 but with removing reaction, verify it updates correctly.

## Deployment Plan
N/A

## Other Considerations
N/A
